airgun/views/oscaptailoringfile.py

```diff
@@ -14,17 +14,19 @@
 from airgun.widgets import (
     ActionsDropdown,
     MultiSelect,
-    SatTable,
+    Table,
 )
 
 
 class SCAPTailoringFilesView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Tailoring Files']")
     new = Text("//a[contains(@href, 'tailoring_files/new')]")
-    table = SatTable(
+    table = Table(
         './/table',
         column_widgets={
-            'Title': Text('./a'),
+            'Name': Text("./a"),
+            'Filename': Text("./a"),
```

Comment:
I'm just curious will this work anytime? because "./a" point to the same locator in a table. Not sure I'm missing anything. you can use examples mentioned here https://github.com/SatelliteQE/airgun/blob/e5ed58aac970094ebd71923cac1451571a9ecced/airgun/views/puppet_environment.py#L35

Reply:
Widget definition in `column_widgets` is created relative to table cell (value in row under column with the same name as `column_widgets` dict key), so `Text("./a")` for Name is different element than `Text("./a")` for Filename.

Of course it's good idea to make widget little more specific where possible, to avoid misunderstandings like this one.

Reply:
Yeah this works but will update this to avoid confusion. Thanks.